app/services/symphony_client.rb

```diff
@@ -64,8 +68,13 @@ def patron_info(patron_key, item_details: {})
       ].join(',')
     })
 
-    JSON.parse(response.body)
+    begin
+      JSON.parse(response.body)
+    rescue StandardError
```

Comment:
Should this be a similar `rescue JSON::ParserError` as above?

Reply:
Ideally, but it was breaking the test because it did not expect a body even if provided. I got it to work , however, by removing `body` which do not seem to be necessary.

Reply:
I pushed up a commit that adds the `.body` back.  I feel like it's a bit more idiomatic and less prone to potential issues down the road.

The test that was failing needed to mimic the `HTTP::Response` interface when stubbing the `authenticated_request` method.